CHANGES.rst

```diff
@@ -389,6 +389,8 @@ Fixed
   compliance with RFC 7232.
 - Fixed ``TypeError`` being raised when using Falcon's testing framework
   to simulate a request to a generator-based WSGI app.
+- Falcon now supports all WebDAV methods (RFC 2518 and RFC 4918), such as
```

Comment:
* We should move this note to the 3.0.0 section. 2.0 has already been released.
* We should also add this note to `3.0.0` RST file.

Reply:
Actually, we need a new `docs/_newsfragment/` for this. I would think this should be classified as a bug fix.

Reply:
Now with the Towncrier setup, yes :slightly_smiling_face: 
Agreed wrt bug fix.